keras/engine/training.py

```diff
@@ -4,12 +4,13 @@
 
 import warnings
 import copy
-import time
 import numpy as np
-import multiprocessing
-import threading
 import six
 
+from keras.utils.data_utils import Sequence
+from keras.utils.data_utils import GeneratorEnqueuer
+from keras.utils.data_utils import OrderedEnqueuer
```

Comment:
Add these 3 classes to `utils/__init__.py` so they can be imported from `utils` by users (internally it doesn't matter)